jenkins/dataproc-custom-images/prepare-custom-leonardo-jupyter-dataproc-image.sh

```diff
@@ -22,7 +22,7 @@ terra_jupyter_r="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-r:2.1.7"
 terra_jupyter_bioconductor="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-bioconductor:2.1.8"
 terra_jupyter_hail="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-hail:1.0.24"
 terra_jupyter_gatk="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:2.2.9"
-terra_jupyter_aou="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:2.1.17"
```

Comment:
please also update terra_jupyter_aou_old values

Reply:
Old value should match to the value we are currently using: https://github.com/all-of-us/workbench/blob/main/api/config/config_prod.json#L13
seems 2.1.15 is still the right value.